utils/stats.js

```diff
@@ -0,0 +1,58 @@
+//Funcion que da las estadisticas
+
+const fetch = require("node-fetch");
+const chalk = require("chalk");
+
+function statsLinks(links) {
+  let functionalLinks = 0;
+  let brokenLinks = 0;
+
+  const promises = [];
+  links.forEach((link) => {
+    const p = fetch(link)
```

Comment:
Define mejor tu constate
```suggestion
    let promiseFetch= fetch(link)
```